settings.gradle

```diff
@@ -30,7 +30,7 @@ project(':wallettool').name = 'bitcoinj-wallettool'
 include 'examples'
 project(':examples').name = 'bitcoinj-examples'
 
-if (GradleVersion.current().compareTo(minFxGradleVersion) > 0) {
+if (GradleVersion.current().compareTo(minFxGradleVersion) > 0 && JavaVersion.current().isCompatibleWith(JavaVersion.VERSION_17)) {
```

Comment:
This is missing another "else if" case. Currently, it can print

> Skipping wallettemplate, requires Gradle Gradle 7.3+, currently running: Gradle 7.5.1

which is misleading because in my case the JDK version is too low.

Reply:
I updated the error message, but didn't add a new case to the conditional, so the output is:

```
Skipping wallettemplate, requires Gradle Gradle 7.3+ and Java 17+, currently running Gradle: Gradle 8.3, Java 11
```